### DISCUSSION TITLE
Projects UI has horizontal and vertical scroll

### DISCUSSION THREAD

**Discussion by raphaelboukara** [community/community#17301] [Projects and Issues]
- Projects UI has horizontal scroll for any king of views (table and board).
- Projects UI got vertical scroll for board view.

Is it intentional?

**Comment by leetrout** [comment]
I'll add that it's not obvious how to scroll horizontally. My mouse usually works with scroll wheel click but no dice.

**Comment by raphaelboukara** [comment]
It seems to be solved, did you deploy a fix?
